### PR TITLE
[GDPR] Adjusted TOS/Privacy acceptance

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Code Changes
- **register.component.html**: Removed old acceptance views. Added new checkbox and label elements. Styled similarly to other helper text within this component. 
- **messages.json**: Added label message and error message for policy acceptance.
- **jslib**: Update jslib from 76c0964 to 5e50aa1

## Screenshots
web-0-base
web-1-error

### CODE REVIEW

File: src/app/accounts/register.component.html [hunk 1]
```diff
@@ -25,7 +25,8 @@ <h2>Companies globally trust Bitwarden for password management.</h2>
                                 </cite>
                             </figcaption>
                             <blockquote>
-                                "Bitwarden has become a popular choice among open-source software advocates. After using it for a
+                                "Bitwarden has become a popular choice among open-source software advocates. After using
+                                it for a
                                 few months, I can see why." - February 2020
```
Comment (cscharf) [inline src/app/accounts/register.component.html:30]: move this line up for continuity in code style.

**Review by cscharf — COMMENTED** [review]
Just one minor formatting nit. Also added GO to review verbiage and final fit/finish.

**Review by go12 — APPROVED** [review]

File: src/app/accounts/register.component.html [hunk 2]
```diff
@@ -132,13 +146,6 @@ <h2>Companies globally trust Bitwarden for password management.</h2>
                                         {{'cancel' | i18n}}
                                     </a>
                                 </div>
-                                <small class="text-muted" *ngIf="showTerms">
-                                    {{'submitAgreePolicies' | i18n}}
```
Comment (kspearrin) [inline src/app/accounts/register.component.html:136]: remove this old string?

**Review by kspearrin — CHANGES_REQUESTED** [review]

File: src/app/accounts/register.component.html (hunk 2 above)
Comment (vincentsalucci) [reply src/app/accounts/register.component.html:136]: Removed!

**Review by kspearrin — DISMISSED** [review]

**Review by kspearrin — APPROVED** [review]

**Pull request merged**